src/main/drivers/pwm_output_stm32f3xx.c

```diff
@@ -169,14 +169,14 @@ void pwmDigitalMotorHardwareConfig(const timerHardware_t *timerHardware, uint8_t
 
     TIM_OCStructInit(&TIM_OCInitStructure);
     TIM_OCInitStructure.TIM_OCMode = TIM_OCMode_PWM1;
-    if (timerHardware->output & TIMER_OUTPUT_N_CHANNEL) {
+    if (output & TIMER_OUTPUT_N_CHANNEL) {
```

Comment:
Someone should review these lines, I don't have real F7 hardware to test on.
Maybe @sambas could help?